src/styles/blocks/catalog.scss

```
@@ -17,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-property: color transform;
+    transition-duration: $transitionDuration;
```

```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```
you can combine it using `transition`

---

Thanks) 
Done